### DISCUSSION TITLE
Validation possible?

### DISCUSSION THREAD

**Discussion by esfel** [rdz-oss/BattyBirdNET-Analyzer#24] [Q&A]
Hi! I've just got BattyBirdNET running - thanks a lot for this great extension of BirdNET. I'm wondering if there's a way to (manually) validate the output (like in BirdNET)?

Some background: I have a large dataset of audio recordings with AudioMoths with a sampling rate of 192 kHz from forest sites in mid-Germany, and was originally only going to analyze birds with BirdNET but just came across BattyBirdNET so now want to include bats as well. I built my workflow on BirdNET which includes manual validation of some detections per species - writing WAV snippets of the detections with R, and then listening to them - and would like to do something similar with BattyBirdNET, I just don't really see how yet. Do you have any ideas?

I'm new to working with bats, also I still get a bit confused about sampling rate / frequencies, but figured out so far that my recordings with 192 kHz sampling rate will include some (but not all) bats. I chose the EU 256 kHz classifier.

Thanks! Esther

**Comment by rdz-oss** [comment]
Hi @esfel ,

hope you have found a good way. Some thoughts:

- at 192kHz you can detect signals to about 192/2 approx 80kHz. That should capture some identifiable calls of several species, actually. You will have issues with myotis and horshoe bats, but you still have likely captured quite a few other species with enough to id them.
- You can use the --spectrum on option of bat_ident.py, this will generate spectrogram .png files for each file for you to look at 
- you can load the segmented files into an analysis software of your liking. You should be able to load them as .wav files into scipts, too. GUANO meta-data has been added to the segments generated.

How did you build your workflow? In python or R ? Or using the BirdNET-Analyzer?
- If you use the BirdNET-Analyzer, it should be able to use BattyBirdNET-Analyzer in a similar way.

Best regards,
Richard

**Reply by rdz-oss** [reply]
you could ask @pete-the-geek how he uses the BTO pipeline in his workflow.

**Reply by Pete-the-Geek** [reply]
Well you could but I don't have any automation with BTO. I simply export the RAW WAV files and run them through BTO pipeline as well as BatDetect2 and Sonobat. BTO is the only package that provides results for some mammals though I also have reference books for numerous mammals that I use to manually identify them. The software then spits out results that I import into my own program and that then allows me to compare the results and decide what is the most likely bat(s)/mammal(s). I then do whatever measurements I need to do using various packages, including Sonobat, Kaleidoscope and BatInspector to confirm what I believe them to be and tag it in my software. I use my software to then export the results and upload them to national records.
